src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -3322,6 +3315,70 @@ private void RecordAvBase(string videoWriterName, string filename, bool unattend
 			RewireSound();
 		}
 
+		private (IVideoProvider, Action/*?*/ Dispose) GetCaptureProvider()

```

Comment:
```suggestion
		private (IVideoProvider Output, Action/*?*/ Dispose) GetCaptureProvider()
```
(And the other build error was from master, fixed now if you would rebase again.)

Reply:
Done